.kitchen.yml

```diff
@@ -24,6 +24,17 @@ verifier:
   fail_fast: false
 
 suites:
+  - name: custom_service_accounts
```

Comment:
Only changes in this file is to order the test fixtures alphabetically, and rename the commands where necessary.